.github/workflows/dependabot_changelog.yml

```diff
@@ -0,0 +1,23 @@
+name: Write changelog for dependabot PR
+on:
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  add-changelog:
+    runs-on: 'ubuntu-latest'
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+      - name: Write, commit and push changelog
+        run: |
+          echo "${{ github.event.pull_request.title }}." > "changelog.d/${{ github.event.pull_request.number }}".docker
```

Comment:
Should this not be `.misc`?

Reply:
Rationale: altering the lockfile changes the contents of the Docker images and Debian packages.

Reply:
It also changes the dependencies for all poetry based installations. Personally I'd read something in `.docker` to be something that *only* affects docker?

Reply:
That's fair. I'll commit directly to `develop` to make it `misc`.

Reply:
Actually I'll be a good boy and raise a PR so that we have a proper changelog.